npm/pod.go

```diff
@@ -140,6 +150,17 @@ func (npMgr *NetworkPolicyManager) UpdatePod(oldPodObj, newPodObj *corev1.Pod) e
 		newPodObjNs, newPodObjName, newPodObjLabel, newPodObjPhase, newPodObjIP,
 	)
 
+	// today K8s does not allow updating HostNetwork flag for an existing Pod. So NPM can safely
+	// check on the oldPodObj for hostNework value
+	if isHostNetworkPod(oldPodObj) {
```

Comment:
It's a nit pick. Do we want to move this logic before line 129, before isInvalidPodUpdate logic? Check whether isHostNetworkPod is quick and we can return early without compare oldPodObj with newPodObj.

Reply:
Done, moved it above